### ISSUE TITLE
ArgumentOufOfRangeException although passed value is valid

### ISSUE THREAD

**Issue by Syzuna** [MarkCiliaVincenti/AsyncKeyedLock#3]
Affected Version: >= 6.0.1 (5.x works)

Stacktrace: 
```
Unhandled exception. System.ArgumentOutOfRangeException: MaxCount should be greater than or equal to 1. (Parameter 'options')
Actual value was 1.
   at AsyncKeyedLock.AsyncKeyedLockDictionary`1..ctor(AsyncKeyedLockOptions options)
   at AsyncKeyedLock.AsyncKeyedLocker`1..ctor(AsyncKeyedLockOptions options)
```

It is obvious from the stacktrace that the check seems to be behaving incorrectly.
The value that was passed is 1 and is greater than or equal to 1 obviously.

Code that leads to the error:
```
new AsyncKeyedLocker<string>(new AsyncKeyedLockOptions(1, 25))
```

**Comment by MarkCiliaVincenti** [comment]
Thank you. The issue has been fixed and is available in 6.0.2.

**Comment by MarkCiliaVincenti** [comment]
Credited as contributor @ https://github.com/MarkCiliaVincenti/AsyncKeyedLock/blob/master/CONTRIBUTORS.md

Thanks again!
